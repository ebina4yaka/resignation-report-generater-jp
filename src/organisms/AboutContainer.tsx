import React from 'react'
import Container from '@material-ui/core/Container'
import { createStyles, makeStyles } from '@material-ui/core/styles'
import { Paper } from '@material-ui/core'
import AboutParameters from '../molecules/AboutParameters'
import AboutUse from '../molecules/AboutUse'

const useStyles = makeStyles((theme) =>
  createStyles({
    root: {
      padding: theme.spacing(2, 0, 2),
      flexDirection: 'column',
      justifyContent: 'center',
    },
    content: {
      padding: theme.spacing(2, 6, 3),
      margin: theme.spacing(2, 4, 2),
    },
  })
)

export default function AboutContainer(): React.ReactElement {
  const classes = useStyles()
  return (
    <Container className={classes.root} maxWidth="md">
      <Paper className={classes.content}>
        <AboutUse />
      </Paper>
      <Paper className={classes.content}>
        <AboutParameters />
      </Paper>
    </Container>
  )
}
