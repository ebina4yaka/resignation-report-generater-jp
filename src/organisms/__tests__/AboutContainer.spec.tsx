import React from 'react'
import renderer from 'react-test-renderer'
import AboutContainer from '../AboutContainer'

jest.mock('../../molecules/AboutParameters', () => 'MockedAboutParameters')
jest.mock('../../molecules/AboutUse', () => 'MockedAboutUse')

test('Snapshot test for AboutContainer', () => {
  const tree = renderer.create(<AboutContainer />).toJSON()
  expect(tree).toMatchSnapshot()
})
