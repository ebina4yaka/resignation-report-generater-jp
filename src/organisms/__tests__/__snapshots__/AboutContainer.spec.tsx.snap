// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for AboutContainer 1`] = `
<div
  className="MuiContainer-root makeStyles-root-1 MuiContainer-maxWidthMd"
>
  <div
    className="MuiPaper-root makeStyles-content-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <MockedAboutUse />
  </div>
  <div
    className="MuiPaper-root makeStyles-content-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <MockedAboutParameters />
  </div>
</div>
`;
